gluten-core/pom.xml

```diff
@@ -173,12 +173,14 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>${protobuf.version}</version>
+      <version>${forked.protobuf.version}</version>
+      <scope>provided</scope>
```

Comment:
We tried to replace this with custom protobuf, but got error (class symbol cannot be found) when building generated protobuf source. Maybe, there is some unknown requirement. So we just keep the original dependency in this module with "provided" scope used. The custom protobuf packed into gluten jar will be used at runtime.